ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/search/FullTextSearchViewImpl.java

```diff
@@ -238,5 +239,18 @@ public void onClick(ClickEvent event) {
             showSelectPathDialog();
           }
         });
+
+    text.addKeyUpHandler(
```

Comment:
Hello, @hkolvenbach!
Thank you for your PR, I have a question.
Can we use [this method](https://github.com/hkolvenbach/che/blob/6757073ce2283eda1a644ff93fc8b18623bd0032/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/search/FullTextSearchViewImpl.java#L143) or change logic of [this method](https://github.com/hkolvenbach/che/blob/6757073ce2283eda1a644ff93fc8b18623bd0032/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/search/FullTextSearchPresenter.java#L152) instead of adding new handler here?

Reply:
good suggestion, i changed it. know it will listen to the enter button to start a search anywhere except the browse and cancel button

Reply:
thank you!